source/includes/component-compatibility-table-node.rst

```diff
@@ -0,0 +1,47 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large
+
+   * - Component
+     - Node.js Driver v6.x
+     - Node.js Driver v5.x
+     - Node.js Driver v4.x
+     - Node.js Driver v3.x
+
+   * - bson
+     - ^6.0.0
+     - ^5.0.0
+     - ^4.0.0
+     - ^1.0.0
+
+   * - bson-ext
+     -
```

Comment:
Q: it's not immediately obvious what a blank cell in the table means. would it be clearer to put some other text there (N/A or a hyphen, for example)? or add other explanatory text elsewhere? 

if you think the blank is clear enough, i'll go with it.

Reply:
After trying a couple different methods, I decided to use a blank space because that is how the other compatibility tables indicate not compatible. Since this is new content, I think it would be best to wait for any feedback.

Reply:
I created a ticket to centralize a more detailed description of this new type of compat table https://jira.mongodb.org/browse/DOCSP-32358, and will probably handle that this week.